docs/user/migration-guide.md

```diff
@@ -0,0 +1,65 @@
+# Migration to v1.0.0-alpha.5
+
+The release includes several breaking changes that should be handled carefully if you are updating Kubeapps from a version prior to v1.0.0-alpha.5. As a summary this release include the following breaking changes:
+
+ - The recommended way of installing Kubeapps is through its helm chart.
+ - The `kubeapps` cli is now deprecated. **It won't be included in future releases**.
+ - Kubeapps no longer includes Tiller, Kubeless nor SealedSecrets by default.
```

Comment:
I think installs might be the better word here. Also I think "Kubeless and SealedSecrets" reads better (instead of nor, which sounds like a double negative because of the "no longer")